modules/alloy/src/main/ts/ephox/alloy/api/ui/SplitFloatingToolbar.ts

```diff
@@ -93,11 +89,7 @@ const factory: CompositeSketchFactory<SplitFloatingToolbarDetail, SplitFloatingT
           FloatingToolbarButton.reposition(floatingToolbarButton);
         });
       },
-      getOverflow: (toolbar: AlloyComponent) => {
-        return memFloatingToolbarButton.getOpt(toolbar).bind((floatingToolbarButton) => {
-          return FloatingToolbarButton.getToolbar(floatingToolbarButton);
-        });
-      }
+      getOverflow: (toolbar: AlloyComponent) => memFloatingToolbarButton.getOpt(toolbar).bind((floatingToolbarButton) => FloatingToolbarButton.getToolbar(floatingToolbarButton))
```

Comment:
One more that's been shoved onto a single line making it harder to read.

Reply:
Fixed